operator/src/main/resources/Operator.properties

```diff
@@ -183,6 +183,9 @@ WLSKO-0170=HTTP timeout: exception {0}.
 WLSKO-0171=Namespace {0} is in operator's target namespaces list but doesn't exist
 WLSKO-0173=Replace custom resource definition failed: {0}
 WLSKO-0174=Create custom resource definition failed: {0}
+WLSKO-0175=Job {1} in namespace {0} failed, job pod name is {2}, and job status is {3}. Check SEVERE log \
```

Comment:
There won't always be SEVERE log messages copied from the introspector pod - for example on an image pull failure.  I'm not sure what's logged in that case...

Reply:
Maybe the log message text should be updated accordingly, and the INFO changed to SEVERE?

Reply:
What about removing SEVERE from the log message and keep the rest?

Reply:
reworded the log message.